easybuild/easyconfigs/f/FIX/FIX-1.068-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Tarball'
+
+name = 'FIX'
+version = '1.068'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX'
+description = """FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad
+components can be removed from the 4D FMRI data."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://www.fmrib.ox.ac.uk/~steve/ftp/']
+sources = ['fix.tar.gz']
```

Comment:
@crubb this tarball points to the latest version.
better to change this to `fix-1.06.12.tar.gz` to avoid that a different version is downloaded in the future.

Reply:
Fixed. Good catch. It's a little troublesome, that the Wiki lists [v1.068 as the lastest version](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX), and there does not seem to be an easy way to get old versions (at least the obvious URLs: 404). 

https://git.fmrib.ox.ac.uk/fsl/fix/-/tags contains the source, but since the `training_files/*` are LFS, they're not included in the archive, which is a known "feature proposal", i.e. bug, with gitlab - https://gitlab.com/gitlab-org/gitlab-foss/issues/14261